modules/account-lib/src/coin/eth/transactionBuilder.ts

```diff
@@ -358,14 +368,23 @@ export class TransactionBuilder extends BaseTransactionBuilder {
     this._counter = counter;
   }
 
+  /**
+   * The value to send along with this transaction. 0 by default
+   *
+   * @param {string} value The value to send along with this transaction
+   */
+  value(value: string): void {
+    this._value = value;
+  }
+

```

Comment:
why exactly do we need to send from a single sig address and it changes this?

Reply:
When sending through the contract, we always send 0 value in the external tx. Otherwise we would be sending from the fee address to the wallet when we are just trying to send from the wallet externally. The actual send value is encoded in `transferBuilder`.